adhearsion-asterisk.gemspec

```diff
@@ -18,16 +18,14 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency %q<adhearsion>, ["~> 2.5"]
-  s.add_runtime_dependency %q<jruby-openssl> if RUBY_PLATFORM == 'java'
+  s.add_runtime_dependency %q<adhearsion>, ["3.0.0.rc1"]
```

Comment:
@gfaza How about `~> 3.0` for adhearsion?

Reply:
bundler couldn't seem to find it for me.